_data/Luca-Castelnuovo/Luca-Castelnuovo.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Luca Castelnuovo
+institution: Het Baarnsch Lyceum
+profile_pic: luca-castelnuovo.jpeg
+quote: DB destroyed because you bought a new server and forgot the data was on the old server -- 10/10 experience
```

Comment:
Character count of your quote exceeds the limit-100 given in the [instructions](https://github.com/education/graduation#3-add-your-profile-information), you might want to modify this